cypress/tools/fixture-generator.py

```diff
@@ -0,0 +1,54 @@
+import json
+import random
+import string
+import uuid
+from datetime import datetime, timedelta
+import os
+
+def random_string(length=8):
+    return ''.join(random.choices(string.printable + 
+                                  string.digits + 
+                                  string.punctuation + 
+                                  string.whitespace, k=length))
```

Comment:
@sergesoroka `string.printable` should be enough, because [it includes the other ones](https://docs.python.org/3/library/string.html#string.printable).